nautobot/core/settings.py

```diff
@@ -1097,17 +1097,14 @@ def silk_request_logging_intercept_logic(request):
 # Kubernetes settings variables
 #
 
-# File path to the config file used to load and configure the kubernetes cluster
-KUBECONFIG = os.getenv("NAUTOBOT_KUBECONFIG", "")
+# Filepath of the kubernetes pod manifest used to create a job pod in the kubernetes cluster
+KUBERNETES_JOB_POD_MANIFEST = os.getenv("NAUTOBOT_KUBERNETES_JOB_POD_MANIFEST", {})
```

Comment:
Update comment appropriately; also this probably isn't doable as an environment variable.

Reply:
Instead of os.getenv, just set it to a dictionary.